faucet/app.py

```diff
@@ -52,6 +52,9 @@ async def transfer(
     Defaults to 1 ether and 35k gas.
     """
     with networks.parse_network_choice(NETWORK_TRIPLE) as provider:
+        # Handle ENS domains by parsing for "."
+        if "." in address:
+            address = convert(address, AddressType)
```

Comment:
I think that `AccountAPI.transfer` does automatic conversion, so this isn't needed if the ens plugin is installed